sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -99,16 +100,6 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
     session.sharedState.cacheManager.uncacheQuery(session, v2Relation, cascade = true)
   }
 
-  private def makeQualifiedDBObjectPath(location: String): String = {
```

Comment:
Exposing this logic in the companion object to be used for staging tables as well.